src/Harald.Infrastructure.Slack/Model/SlackChannelName.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Harald.Infrastructure.Slack.Model
+{
+    public class SlackChannelName : IValidatableObject
+    {
+        private readonly string _value;
+
+        public SlackChannelName(string value)
+        {
+            _value = value;
+        }
+
+        public override string ToString()
+        {
+            return _value;
+        }
+
+        public static implicit operator SlackChannelName(string input)
+        {
+            return new SlackChannelName(input);
+        }
+
+        public static implicit operator string(SlackChannelName input)
+        {
+            return input.ToString();
+        }
+
+        public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+        {
+            var results = new List<ValidationResult>();
+
+            if (_value.Length > 21)
```

Comment:
[(#30) 80 char slack channel name](https://github.com/dfds/harald/pull/30)

Reply:
I have changed this to support 80 chars or less. :)